qiskit_experiments/library/tomography/fitters/lstsq_utils.py

```diff
@@ -189,7 +189,13 @@ def zero(_):
     return basis_mat, probs, prob_weights
 
 
-@deprecate_arguments({"beta": "outcome_prior"})
+@deprecate_arg(
+    "beta",
+    new_alias="outcome_prior",
+    since="0.5",
+    additional_msg="Instead, use the ``outcome_prior`` argument.",
```

Comment:
This message is automatically generated.
https://github.com/Qiskit/qiskit-terra/blob/364d6fdea34c7ec54e6f5d3581e8a0ec6c8bfb5d/qiskit/utils/deprecation.py#L151-L152

Reply:
Thanks for catching this, I've removed it.